07-data-models-new-field/README.md

```diff
@@ -145,23 +145,26 @@ Now you should see an output similar to the below:
 
 ```json
 {
-  "created": "2019-03-13T10:39:57.345889+00:00",
-  "id": "2",
+  "_bucket": "60f2b083-8f7b-4aba-a00e-09e3bb3e12af",
+  "created": "2019-11-25T07:41:44.620275+00:00",
+  "id": "1",
   "links": {
-    "self": "https://localhost:5000/api/records/2"
+    "files": "https://localhost:5000/api/records/1/files",
+    "self": "https://localhost:5000/api/records/1"
   },
   "metadata": {
+    "$schema": "https://my-site.com/schemas/records/record-v1.0.0.json",
```

Comment:
moderate: Is not the fault of the work, maybe Invenio is not fully working. Is this supposed to be in the output?

Reply:
This was changed, in my pr #71